acceptance/telemetry/upload-fails/script

```diff
@@ -0,0 +1,12 @@
+export DATABRICKS_CLI_TELEMETRY_PID_FILE=./telemetry.pid
+export DATABRICKS_CLI_TELEMETRY_UPLOAD_LOGS_FILE=./out.upload_process.txt
+
+# This test ensures that the main CLI command does not error even if
+# telemetry upload fails.
+trace $CLI selftest send-telemetry
+
+# Wait for the child telemetry process to finish
+wait_pid.py $(cat ./telemetry.pid)
+
+# cleanup the pid file
+rm -f ./telemetry.pid
```

Comment:
If you always expect telemtry.pid here, then 'rm telemetry.pid' (without -f) acts as an additional assertion that it was there.

Reply:
Good point, removed the `-f` flag.